evaluation/evaluation_pipeline/big_query_helper.py

```diff
@@ -38,13 +43,15 @@ def insert_rows(self, table_name, rows):
     table_ref = self._dataset.table(table_name)
     try:
       table = self._client.get_table(table_ref)
-      errors = self._client.insert_rows(table, rows)
-      if errors:
-        for e in errors:
-          logging.error('Failure when inserting row: %s', str(e))
-      else:
-        logging.info('Successfully inserted %d rows into %s', len(rows),
-                     table_name)
+
+      for row_chunk in chunk_rows(rows):
```

Comment:
maybe add a function that takes a chunk and tries to insert it and inline the `chunk_rows` instead? Up to you.

Reply:
Sounds good. Will update.